site/en/docs/privacy-sandbox/fedcm/index.md

```diff
@@ -216,26 +242,598 @@ They can do the same for Chrome on desktop by going to
+}
+```
+
+The returned client metadata is consumed by the browser and will not be
+available to the RP.
+
+#### ID assertion endpoint {: #id-assertion-endpoint }
+
+The IdP's ID assertion endpoint returns an assertion for their signed-in user.
+When the user signs in to an RP website using [`navigator.credentials.get()`
+call](#sign-into-rp), the browser sends a `POST` request with cookies and a
+content type of `application/x-www-form-urlencoded` to this endpoint with the
+following information:
+
+<table class="with-heading-tint with-borders">
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tr>
+    <td><code>client_id</code> (required)</td>
+    <td>The RP's client identifier.</td>
+  </tr>
+  <tr>
+     <td><code>account_id</code> (required)</td>
+     <td>The unique ID of the signing in user.</td>
+  </tr>
+  <tr>
+     <td><code>nonce</code> (optional)</td>
+     <td>The request nonce, provided by the RP.</td>
+  </tr>
+  <tr>
+     <td><code>disclosure_text_shown</code></td>
+     <td>Results in a string of <code>"true"</code> or <code>"false"</code> (rather than a boolean). The result is <code>"false"</code> if the disclosure text was not shown. This happens when the RP's client ID was included in the <code>approved_clients</code> property list of the response from the <a href="#accounts-list-endpoint">accounts list endpoint</a>.</td>
+  </tr>
+</table>
+
+Example HTTP header:
+
+```http
+POST /id_assertions.php HTTP/1.1
+Host: idp.example
+Referer: https://rp.example/
+Content-Type: application/x-www-form-urlencoded
+Cookie: 0x23223
+Sec-Fetch-Dest: webidentity
+account_id=123&client_id=client1234&nonce=Ct60bD&disclosure_text_shown=true
+```
+
+On the server, the IdP should confirm that:
+
+1. The claimed account ID matches the ID for the account that is already
+   signed in. 
+2. The `Referer` header matches the origin the RP, registered in advance
+   for the given client ID.
+
+{% Aside 'warning' %}
+
+Since the domain verification on OAuth or OpenID Connect relies on a browser
+redirect, it's critical in FedCM that the IdP server checks a `Referer` header value
+matches the RP's registered origin.
+
+{% endAside %}
+
+The browser expects a JSON response that includes the following property:
+
+<table class="with-heading-tint with-borders">
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tr>
+    <td><code>token</code> (required)</td>
+    <td>A token is a string that contains claims about the authentication.</td>
+  </tr>
+</table>
+
+```json
+{
+  "token": "eyJ********"
+}
+```
+
+The returned token is passed to the RP by the browser, so that the RP can
+validate the authentication.
+
+### Sign in to the identity provider from the relying party {: #sign-into-rp }
+
+Once the IdP's configuration and endpoints are available, RPs can call
+`navigator.credentials.get()` to request allowing users to sign in to the IdP
```

Comment:
Why is this "sign in to the IdP from the RP"? I suppose you meant "allowing users to sign in to the RP with the IdP"? Same for the title above at line 718.

Reply:
Not sure why. Fixed. Thanks for the catch!